services/web-server/src/login/strategies/mozilla-auth0.js

```diff
@@ -168,8 +168,8 @@ module.exports = class MozillaAuth0 {
     // For our own purposes, if the prefix is not mozilliansorg. then we treat it as an ldap group
```

Comment:
I think this comment is outdated, too, as we aren't looking at prefixes anymore.  I think [this schema](https://auth.mozilla.com/.well-known/profile.schema) defines the access_information property.  If so, I'll fix the comment.

Reply:
Agreed.